soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10978,7 +10978,7 @@ void Player_UpdateCommon(Player* this, GlobalContext* globalCtx, Input* input) {
     Collider_ResetQuadAC(globalCtx, &this->shieldQuad.base);
     Collider_ResetQuadAT(globalCtx, &this->shieldQuad.base);
 
-    if (this->iceTrapped != 0) {
+    if (this->pendingIceTrap == true) {
```

Comment:
no need to have `== true` here, just `if (this->pendingIceTrap)` works

Reply:
Woops.